.github/ISSUE_TEMPLATE/feature_request.yaml

```
@@ -0,0 +1,48 @@
+name: Feature request
+description: Suggest a new feature for MSAL.NET.
+labels: ["feature request", "untriaged", "needs attention"]
+title : '[Feature Request]     '
+body:
+- type: markdown
+  attributes:
+    value: |
+      ## Before submitting your feature request
+      Please make sure that your question or issue is not already covered in [MSAL documentation](https://learn.microsoft.com/entra/msal/dotnet/) or [samples](https://learn.microsoft.com/azure/active-directory/develop/sample-v2-code?tabs=apptype).
+      
+- type: markdown
+  attributes:
+    value: |
+      ## Feature request for MSAL.NET
+      
+- type: dropdown
+  attributes:
+    label: MSAL client type
+    description: Are you using PublicClientApplication (desktop and mobile apps), ConfidentialClientApplication (web apps, web APIs, service-to-service) or ManagedIdentityApplication?
+    multiple: true
+    options: 
+      - "Public"
+      - "Confidential"
+      - "Managed identity"
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Description
```

How about this?

```suggestion
    label: Problem Statement
```

Otherwise, this PR LGTM.